cmd/nerdctl/image_encrypt_linux_test.go

```diff
@@ -68,7 +68,7 @@ func newJWEKeyPair(t testing.TB) *jweKeyPair {
 func rmiAll(base *testutil.Base) {
 	base.T.Logf("Pruning images")
 	imageIDs := base.Cmd("images", "--no-trunc", "-a", "-q").OutLines()
-	base.Cmd(append([]string{"rmi", "-f"}, imageIDs...)...).AssertOK()
+	base.Cmd(append([]string{"rmi", "-f"}, imageIDs...)...).Run()
```

Comment:
This change is because when calling `rmi -f allIDs`, tests cannot ensure no running containers. (e.g., in `TestImageEncryptJWE` there is always a registry container running, causing this error:

```
conflict: unable to delete sha256:5bb9b919833aa955dfe1d1121cc038330b025ec6506ce47066c9192927e3dc3d (cannot be forced) - image is being used by running container c1b56cc721c5d1717f842ca7cc3277122e09cb964fcabca22b4bbeea4d57880a
```

Reply:
added